packages/kbn-pm/package.json

```diff
@@ -48,6 +48,7 @@
     "globby": "^8.0.1",
     "has-ansi": "^3.0.0",
     "indent-string": "^3.2.0",
+    "is-path-inside": "^3.0.2",
```

Comment:
Is a dependency really necessary to test if a path starts with another path?

Reply:
Sorry, didn't see this before merge. It's not necessary but it covers some edge cases, is already installed (checkout the changes to yarn.lock) and isn't included in the front end so I didn't see any harm in it.

Reply:
It's also hyper focused so it would be really easy to rip out, and will be really easy to upgrade. It is the best kind of dependency if you ask me.

Reply:
That's fair. Though looks like the only edge case is with 32bit Windows which we don't support.

This really should be part `path` or something in core utils.